src/selectors/prescription.js

```diff
@@ -69,13 +69,21 @@ export const selectFilteredAndSortedItems = createSelector(
   }
 );
 
-export const selectSelectedRows = ({ prescription }) => {
-  const { transaction } = prescription;
-  const { items = [] } = transaction || {};
-
-  return items.reduce((acc, { item }) => ({ ...acc, [item.id]: true }), {});
+export const selectNumberOfItems = ({ prescription }) => {
```

Comment:
I'm missing the point of introducing this selector. Is it just to help the `createSelector` prevent re-renders?

Reply:
yeah - generally we work with `Realm.Results` objects which have funny ways it works with references. i.e if you reference `Transaction.items` it will be a new reference and re-render every time. 

This helps to only create a new reference if the number of items in the result is different. Doesn't help if you actually edit the quantity of an item in the result, only for adding or removing, which is what we want to know about for determining if a row should be highlighted

Reply:
ah, I see. nice!